### ISSUE TITLE
hasGlobalPhantom() doesn't work for node.js 0.10.22

### ISSUE THREAD

**Issue by leochen1216** [dflynn15/gulp-jasmine-phantom#46]
https://github.com/dflynn15/gulp-jasmine-phantom/blob/master/index.js#L9

```
require('child_process').execSync  // is undefined in 0.10.22
```

**Comment by dflynn15** [comment]
Yep. It is in the README
> This module uses execSync which is not available in any version of Node under `0.12.x`

I don't have any plans of managing support for `0.10.x` at the moment. If this is something that you want to see and cannot upgrade I can look at alternatives, but for now it should be a pretty forward upgrade.

**Comment by leochen1216** [comment]
hmm, thanks for your reply. maybe we should update the package.json to be `"node": ">=0.10.0"`?
https://github.com/dflynn15/gulp-jasmine-phantom/blob/master/package.json#L8
